mariadb/rootfs/etc/s6-overlay/s6-rc.d/mariadb-post/run

```diff
@@ -1,32 +1,26 @@
-#!/usr/bin/with-contenv bashio
+#!/command/with-contenv bashio
+# shellcheck shell=bash
 # ==============================================================================
-# Start MariaDB service if enabled
+# Post-start initialization of MariaDB service
 # ==============================================================================
-MARIADB_DATA=/data/databases
-NEW_INSTALL=false
 
-# Init mariadb
-if ! bashio::fs.directory_exists "${MARIADB_DATA}"; then
-    bashio::log.info "Create a new mariadb initial system"
-    mysql_install_db --user=root --datadir="${MARIADB_DATA}" --skip-name-resolve --skip-test-db > /dev/null
-    NEW_INSTALL=true
-else
-    bashio::log.info "Using existing mariadb initial system"
-fi
-
-# Redirect log output
-rm -f "${MARIADB_DATA}/mariadb.err"
-ln -s /proc/1/fd/1 "${MARIADB_DATA}/mariadb.err"
-
-# Start mariadb
-bashio::log.info "Starting MariaDB"
-mysqld_safe --datadir="${MARIADB_DATA}" --user=root < /dev/null &
-MARIADB_PID=$!
+function execute_without_error_messages() {
+    local current_log_level="${__BASHIO_LOG_LEVELS[${__BASHIO_LOG_LEVEL}]}"
+    bashio::log.level fatal
+    local exit_code=0
+    $@ || exit_code=$?
+    bashio::log.level ${current_log_level}
+    return ${exit_code}
+}
 
 # Wait until DB is running
 while ! mysql -e "" 2> /dev/null; do
     sleep 1
 done
+bashio::log.info "MariaDB started"
+
+bashio::log.info "Deleting service information from Home Assistant"
+execute_without_error_messages bashio::services.delete "mysql" || true
```

Comment:
Is this required with this changes in? From what I understand the finish script is always called, so this shouldn't be necessary, unless for some reason the API wasn't available on shutdown but is again now?

I guess it also doesn't hurt so :man_shrugging:

Reply:
This is just a failsafe trick, this is for the possibility, that the finish script wasn't called, and that can be caused eg. by a manual `docker kill` command.

Reply:
Maybe remove the log message, just do it silently, no harm, and no questions about the log message.

Reply:
Yes, let's do that.